### PR TITLE
Mocksim udpate

### PR DESCRIPTION
**Pull request by jbae11**
Adds an additional `MockSim::DummyProto` so that the user can define the capacity and commodity name of dummy prototype

### CODE REVIEW

File: src/mock_sim.cc [hunk 1]
```diff
@@ -176,6 +176,20 @@ void MockSim::DummyProto(std::string name) {
   ctx_.AddPrototype(name, a);
 };
 
+void MockSim::DummyProto(std::string name, std::string commod, double capacity) {
```
Comment (scopatz) [inline src/mock_sim.cc]: Doesn't this need to be declared in the header? Also where is it being called?

**Comment by scopatz** [comment]
Thanks for putting this in @jbae11! Please add a news entry, as per http://fuelcycle.org/kernel/contributing_to_cyclus.html#changelog

**Comment by jbae11** [comment]
I forgot to push the header file, my bad. Also, I don't see the `news/` directory. Is that in the cyclus directory?

**Comment by scopatz** [comment]
The news directory is in the root level of the repo.  Maybe you need to update your branch from upstream master?

**Comment by jbae11** [comment]
ah, it's updated! thanks

File: src/mock_sim.cc [hunk 2]
```diff
@@ -176,6 +176,20 @@ void MockSim::DummyProto(std::string name) {
...
+  Agent* a = DynamicModule::Make(&ctx_, AgentSpec(":agents:Source"));
+
+  std::stringstream xml;
+  xml << "<facility><name>" << name << "</name>"
+      << "<config><foo>"
+      << "<commod>" << "commod" << "</commod>"
+      << "<capacity>" << capcity << "</capacity>"
```
Comment (scopatz) [inline src/mock_sim.cc]: Looks like there is a typo here.  Middle one should be `capacity`

**Comment by jbae11** [comment]
so sorry about that, I will make sure it builds next time!

**Comment by scopatz** [comment]
Thanks @jbae11!

**Pull request merged**